src/resolveCallSiteSourceCodeLocation.js

```diff
@@ -12,6 +15,31 @@ import type {
   SourceCodeLocationType,
 } from './types';
 
+const readFile = promisify(fs.readFile);
+
+const cachedSourceMaps: { [string]: Promise<BasicSourceMapConsumer> | typeof undefined, ... } = {};
+
+const resolveOriginalPosition = async (mapFilePath: string, column: number, line: number): Promise<NullableMappedPosition> => {
+  let sourceMapResult = cachedSourceMaps[mapFilePath];
+
+  if (!sourceMapResult) {
+    sourceMapResult = (async () => {
+      const rawSourceMap = JSON.parse(await readFile(mapFilePath, 'utf8'));
+
+      return new SourceMapConsumer(rawSourceMap);
+    })();
+
+    cachedSourceMaps[mapFilePath] = sourceMapResult;
+  }
+
+  const sourceMap = await sourceMapResult;
```

Comment:
The coverage result seems definitely wrong to me, these lines are 100% covered by tests